src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black, White, Green, Pink, Yellow, Blue, Orange
```

Comment:
This violates checklist item #15: 'Name constants in uppercase with underscores, as per the linked Java style guide (also applies to enum constant naming).'

Enum constants should be named in uppercase (e.g., BLACK, WHITE, GREEN, etc.).

Reply:
fix comment (apply for other values)
```suggestion
    BLACK, WHITE, Green, Pink, Yellow, Blue, Orange
```